test/WebSites/FilesWebSite/Controllers/DownloadFilesController.cs

```diff
@@ -17,19 +17,19 @@ public DownloadFilesController(IHostingEnvironment hostingEnvironment)
             _hostingEnvironment = hostingEnvironment;
         }
 
-        public IActionResult DowloadFromDisk()
+        public IActionResult DownloadFromDisk()
```

Comment:
Oh duh just saw the other comments here and https://github.com/aspnet/Mvc/pull/5278...